lib/matplotlib/cbook/__init__.py

```diff
@@ -2231,3 +2223,20 @@ def _picklable_class_constructor(mixin_class, fmt, attr_name, base_class):
     factory = _make_class_factory(mixin_class, fmt, attr_name)
     cls = factory(base_class)
     return cls.__new__(cls)
+
+
+def _unpack_to_numpy(x):
+    """Internal helper to extract data from e.g. pandas and xarray objects."""
```

Comment:
Please document what we intend to support, i.e. everything with .to_numpy() or .values, and what types we expect to catch with it, e.g. values-> older pandas dataframes(?).